version.sbt

```diff
@@ -1 +1 @@
-version in ThisBuild := "0.10.2-SNAPSHOT"
+version in ThisBuild := "0.10.2"
```

Comment:
Not sure if we want to remove the -SNAPSHOT

Reply:
no, that shouldn't be the case. Looks like a FU from last deploy.... Good catch